site/content/en/docs/tasks/setup_multikueue.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Setup a MultiKueue environment"
+date: 2024-02-26
+weight: 9
+description: >
+  Additional steps needed to setup the multikueue clusters.
+---
+
+Check the concepts section for a [MultiKueue overview](/docs/concepts/multikueue/). 
```

Comment:
This tutorial explains how you can configure one management cluster and one worker cluster to run JobSets (link to jobset).